### DISCUSSION TITLE
Are there any plans to support UE5.2?

### DISCUSSION THREAD

**Discussion by yukioshima** [tuatec/TTToolbox#45] [General]
Are there any plans to support UE5.2?

**Comment by tariqaneeq325** [comment]
lets hope they reply

**Comment by b7r3333** [comment]
كولي تبن في ثلاثاء، 10 أكتوبر، 2023 في 3:04 ص، كتب tariqaneeq325 ***@***.***>:
lets hope they reply

**Comment by b7r3333** [comment]
جواهر قحبة هداج والفارس

**Comment by b7r3333** [comment]
جواهر قحبة
